hack/update-generated-protobuf.sh

```diff
@@ -31,19 +31,13 @@ function prereqs() {
   fi
   kube::build::ensure_docker_daemon_connectivity || return 1
 
-  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}/go-to-protobuf")
+  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}")
```

Comment:
Yeah - I think it's possible that I don't understand correctly the purpose of this hash.
So basically, I introduced this hash to avoid situation when running update-generated-protobuf.sh and some other script running on docker at the same time on the same machine will cause some conflicts.
Basically, we are using the same hash here:
https://github.com/kubernetes/kubernetes/blob/master/build/common.sh#L159

but now I can't find any code that may cause problems with that. So either something changed in the meantime or I misunderstood something or I missing something now :)

Anyway - I think we can try with this :)